docs/userguide/layoutengine/index.md

```diff
@@ -116,6 +115,7 @@ the LayoutEngineFactory method to create an instance of the LayoutEngine class
 that is specific to the text. The following demonstrates a call to the
 LayoutEngineFactory:
 
+```c
 LEFontInstace \*font = <the text's font>;
```

Comment:
Shouldn't this be 'LEFontInstace *font = <the text's font>;'? I.e., we should remove the escaping that the auto converter added here and elsewhere in codeblocks.